lib/index.d.ts

```diff
@@ -513,3 +513,12 @@ export class RateLimiterValkeyGlide extends RateLimiterStoreAbstract {
      */
     close(): Promise<void>;
 }
+
+/**
+ * Etcd Rate Limiter class.
+ *
+ * The option "storeClient" needs to be set to an instance of class "EtcdClient".
+ */
+export class RateLimiterEtcd extends RateLimiterStoreAbstract {
```

Comment:
@Tobias4872 `RateLimiterEtcdNonAtomic` Typescript class should be added here as well.

Reply:
@Tobias4872 Should it be like this
```js
export class RateLimiterEtcd extends RateLimiterEtcdNonAtomic
```
RateLimiterEtcd extends RateLimiterEtcdNonAtomic